worldedit-core/src/main/java/com/sk89q/worldedit/internal/command/CommandUtil.java

```diff
@@ -229,7 +229,9 @@ private static Substring onlyOnLastQuotedWord(Substring lastArg, Substring sugge
             return suggestion;
         }
         String substr = suggestion.getSubstring();
-        int sp = substr.lastIndexOf(' ');
+        // Ignore if suggestion ends with a " ", since that signals the end of
+        // the completed command.
+        int sp = substr.trim().lastIndexOf(' ');
```

Comment:
This change makes the code no longer make sense. It’s no longer a check for if it ends with a space, as it no longer can due to being trimmed

Reply:
Oh okay. I probably misunderstood what this mechanism is doing, then...
I thought its purpose was remapping a suggestion such as: `oak slab` to `slab`.
Or in other words: Only suggest starting from the last space within the argument.

At least I noticed that suggestions always start from the last space, no matter whether I'm still in the same argument:

https://user-images.githubusercontent.com/820845/201366233-4611e503-e227-49e0-981f-2934d57bf408.mp4

It's also the way I had to implement it in my plugin, because the Bukkit API only allows you to suggest starting from the last space.

Reply:
I actually think this is correct, though the comment needs adjusting.